common/thorhelper/thorxmlwrite.hpp

```diff
@@ -41,6 +41,8 @@ interface IXmlWriterExt : extends IXmlWriter
     virtual IInterface *saveLocation() const = 0;
     virtual void rewindTo(IInterface *location) = 0;
     virtual void outputNumericString(const char *field, const char *fieldname) = 0;
+    virtual size32_t checkCloseLength(unsigned len, const char* data) = 0;
+    virtual void outputVerbatim(unsigned len, const char* data) = 0;
```

Comment:
I think outputInline() or something similar would be a better name.  This is basically a format neutral version of the existing "outputInlineXml()" function.